Index: .well-known/security.txt
===================================================================
--- .well-known/security.txt	(revision 26916)
+++ .well-known/security.txt	(nonexistent)
@@ -1,3 +0,0 @@
-Contact: https://www.resourcespace.com/bug_report
-Expires: 2025-06-30T22:59:00.000Z
-Policy: https://www.resourcespace.com/bug_report
Index: include/config.default.php
===================================================================
--- include/config.default.php	(revision 26916)
+++ include/config.default.php	(working copy)
@@ -467,7 +467,7 @@
 * MP4: $ffmpeg_preview_options = '-f mp4 -ar 22050 -b 650k -ab 32k -ac 1';
 */
 $ffmpeg_preview=true; 
-$ffmpeg_preview_seconds=120; # how many seconds to preview
+$ffmpeg_preview_seconds=10000; # how many seconds to preview
 $ffmpeg_preview_extension="flv";
 $ffmpeg_preview_min_width=32;
 $ffmpeg_preview_min_height=18;
Index: include/database_functions.php
===================================================================
--- include/database_functions.php	(revision 26916)
+++ include/database_functions.php	(working copy)
@@ -706,6 +706,7 @@
             if(!file_exists($storagedir . "/tmp"))
                 {
                 mkdir($storagedir . "/tmp", 0777, true);
+				
                 }
 
             if(!file_exists($cache_location))
@@ -717,6 +718,7 @@
             }
         catch(Exception $e)
             {
+			debug("storagedir = " . $storagedir);
             debug("SQL_CACHE: {$e->getMessage()}");
             }
         unset($GLOBALS["use_error_exception"]);
Index: pages/tools/staticsync.php
===================================================================
--- pages/tools/staticsync.php	(revision 26916)
+++ pages/tools/staticsync.php	(working copy)
@@ -590,7 +590,7 @@
                                         {
                                         # Save the value
                                         $value = $path_parts[$level-1];
-                                        $modifiedval = hook('staticsync_mapvalue','',array($r, $value));
+                                        $modifiedval = hook('staticsync_mapvalue','',array($r, $value, $field));
                                         if($modifiedval)
                                             {
                                             $value = $modifiedval;
